doc/changelog.rst

```diff
@@ -2,6 +2,61 @@ Changelog
 =========
 
 
+.. _changelog-v2-7-0-rc1:
+
+
+v2.7.0rc1
+---------
+Highlights
```

Comment:
Higlights have to be added manually below this section

Reply:
yep I shall, will put them either in rc2 or (if all goes heavenly) the stable w/o needing rc2

Reply:
then what do you need this pull request for if you have pre-released this on GitHub already?

Reply:
bookkeeping, just bookkeeping, will change it to rc2/stable on it once it's merged

Reply:
docs actually build, see [builds](https://readthedocs.org/projects/esmvalcore/builds/) think the badge is picking up a former fail for some reason

Reply:
yeah, darn badge is picked up from esmvaltool docs builds instead of core :man_facepalming: